external-providers/java-external-provider/pkg/java_external_provider/provider.go

```diff
@@ -296,6 +308,7 @@ func (p *javaProvider) Init(ctx context.Context, log logr.Logger, config provide
 		"-Djava.net.useSystemProxies=true",
 		"-configuration",
 		"./",
+		//"--jvm-arg=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:1044",
```

Comment:
Uncommenting this line allows for the LS to be run in debugging mode. I find it useful, but can be removed if you guys thing it's not convenient.

Reply:
nope that is useful! assume that allows you to attach a debugger and set break points and stuff?